File: 49. Group Anagrams.ts
function groupAnagrams(strs: string[]): string[][] {
const map=new Map();
for (let str of strs)  {
let s=str.split("").sort().join("");
map.has(s) ? map.set(s,[...map.get(s), str] ): map.set(s,[str]);
}
return [...map.values()];
};  

File: 49. Group Anagrams.py
class Solution(object):
    def groupAnagrams(self, strs):
        maps = {}
        for str in strs:
            s = ''.join(sorted(str))
            if s in maps:
                maps[s].append(str)
            else:
                maps[s] = [str]
        return list(maps.values())
        
